src/bcd/bcd/Denoiser.cpp

```diff
@@ -271,15 +271,12 @@ void Denoiser::reorderPixelSetJumpNextChunk(
     }
 }
 
-void reorderPixelSetShuffleCPP11(vector<PixelPosition>& io_rPixelSet)
-{
-    unsigned seed = static_cast<unsigned>(std::chrono::system_clock::now().time_since_epoch().count());
-    shuffle(io_rPixelSet.begin(), io_rPixelSet.end(), std::default_random_engine(seed));
-}
-
 void Denoiser::reorderPixelSetShuffle(vector<PixelPosition>& io_rPixelSet)
 {
-    reorderPixelSetShuffleCPP11(io_rPixelSet);
+    unsigned seed =
```

Comment:
I would prefer if we used a fixed seed so that everything remains deterministic. Would that be possible?

Reply:
I'm not sure about it, but we never enable the random pixel order in appleseed.

Reply:
Ah you mean that this code is not used? Ok then.